consensus/ibft/src/test/java/tech/pegasys/pantheon/consensus/ibft/TestHelpers.java

```diff
@@ -45,4 +61,53 @@ public static Block createProposalBlock(final List<Address> validators, final in
             .setBlockHashFunction(IbftBlockHashing::calculateDataHashForCommittedSeal);
     return new BlockDataGenerator().block(blockOptions);
   }
+
+  public static SignedData<ProposalPayload> createSignedProposalPayload(final KeyPair signerKeys) {
```

Comment:
Are these used outside of the new tests? Generally they are too simple for use in most other tests (which depend upon the msgs content).

Reply:
They are only used in the new tests, but previous comment from @jframe suggested (https://github.com/PegaSysEng/pantheon/pull/501#discussion_r245160869) they be moved to the common helper class as he thought they would be useful in other tests. Thus they are moved.